src/main/scala/io/opentargets/etl/backend/OtarProject.scala

```diff
@@ -31,6 +33,7 @@ object OtarProject extends LazyLogging {
             col("otar_code").as("otar_code"),
             col("project_status").as("status"),
             col("project_name").as("project_name"),
+            col("integrates_data_PPP_Boolean").as("integrates_in_PPP"),
             concat(lit("http://home.opentargets.org/"), col("otar_code")).as("reference")
```

Comment:
I think this can all be done in line 36: 

Something like
```
col("integrates_data_PPP").cast(BooleanType).as("isIntegrated"),
```
The change on line 26 would then be deleted.